app/components/file/browse/display/tree-view-display.component.ts

```diff
@@ -0,0 +1,228 @@
+import { Component, Input, OnInit, ViewChild } from "@angular/core";
+import { Router } from "@angular/router";
+import { TREE_ACTIONS, TreeComponent, TreeModel, TreeNode } from "angular-tree-component";
```

Comment:
add a new-line between the package imports and the application component imports. separation makes it easier to read.